client/src/hooks/useFetch.ts

```diff
@@ -0,0 +1,28 @@
+import { get } from '@/service';
+import { useState, useEffect } from 'react';
+
+interface FetchType {
+  url: string;
+  params?: Record<string, string>;
+}
+
+function useFetch<T>({ url, params }: FetchType) {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState<T>({} as T);
+
+  let requestURL = url;
+  if (params) {
+    requestURL += new URLSearchParams(params).toString();
+  }
```

Comment:
```suggestion
const requestURL = url + params ? new URLSearchParams(params).toString() : ""
``` 

이런식으로 써도 괜찮을 것 같아요! requestURL은 굳이 재할당을 안 해줘도 된다고 생각이 들어서요!

Reply:
오 이방식이 재할당을 안해도 될것 같아서 좋은것 같아요!! 반영하겠습니다~